src/map.cpp

```diff
@@ -684,9 +690,11 @@ void Map::updateLighting(enum LightBank bank,
 
 	//std::map<v3s16, MapBlock*> blocks_to_update;
 
-	std::set<v3s16> light_sources;
+	std::vector<v3s16> light_sources;
+	light_sources.reserve(100);
```

Comment:
as this value us used multiple times now I suggest creating a const variable or a define placing the explanation for it's actual value there too